### PR TITLE
Added Generated Code | Service Version 2020-10-02

### PR DESCRIPTION
**Pull request by mohsha-msft**

### CODE REVIEW

**Review by zezha-msft — COMMENTED** [review]
Wait, the destination branch is not the right one I believe?

File: azblob/access_conditions.go [hunk 1]
```diff
@@ -13,18 +13,18 @@ type ModifiedAccessConditions struct {
 }
 
 // pointers is for internal infrastructure. It returns the fields as pointers.
-func (ac ModifiedAccessConditions) pointers() (ims *time.Time, ius *time.Time, ime *ETag, inme *ETag) {
+func (ac ModifiedAccessConditions) pointers() (ims *time.Time, ius *time.Time, ime *string, inme *string) {
```
[the remark this answers is not in the document]
Comment (mohsha-msft) [reply azblob/access_conditions.go:16]: No, this is internal function and works as expected.

**Comment by mohsha-msft** [comment]
> Wait, the destination branch is not the right one I believe?

Destination branch is not dev?

**Comment by zezha-msft** [comment]
@mohsha-msft sorry I should've been more explicit, the branch should be `adreed/version-level-worm`

**Comment by zezha-msft** [comment]
@mohsha-msft could you please add a test for listing with the option`deletedwithversions` to make sure it works? Thanks

**Comment by mohsha-msft** [comment]
> @mohsha-msft sorry I should've been more explicit, the branch should be `adreed/version-level-worm`

Done and resolved the conflicts.

**Comment by mohsha-msft** [comment]
> @mohsha-msft could you please add a test for listing with the option`deletedwithversions` to make sure it works? Thanks

Done. I've written the test for `DeletedWithVersion`.

**Review by zezha-msft — APPROVED** [review]

**Pull request merged**